Resources/Prototypes/Nyanotrasen/Entities/Structures/Furniture/Barrels/barrels.yml

```diff
@@ -0,0 +1,62 @@
+- type: entity
+  id: BarrelBase
+  parent: BaseStructure
```

Comment:
BaseStructure's physical bounding box is a big square. A circle might not be too bad for this, or perhaps a rectangular shape.

It should be slower to move than the water/fuel tank, since these have no wheels.